scripts/generatePackage.ps1

```diff
@@ -44,7 +44,8 @@ function Update-Package-Install-Links {
 
 function Get-Is-Version-Promoted {
   param ($versionNumber, $packageName)
-  $promotedPackageVersions = (npx sfdx force:package:version:list --released --packages $packageName --json | ConvertFrom-Json).result | Select-Object -ExpandProperty Version
+  $promotedPackageVersions = (npx sf package version list --released --packages $packageName --json | ConvertFrom-Json).result | Select-Object -ExpandProperty Version # ERROR: Unable to convert this command; you must convert it manually.
+
```

Comment:
not sure what this comment is about - `sf package version list --released --packages apex-rollup --json` works just fine

Reply:
Ah, I didn't see the comment. That's from the conversion script, telling me to check the script because it didn't like something.